### PR TITLE
implemented homework 1-1

### PR DESCRIPTION
**Pull request by ivanzdravchev-acc**

### CODE REVIEW

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (yll090) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Please try to remove debugging/stray comments out of the PR when possible.

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (yll090) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: You can extract the resource ID & content description as variables based on the condition and use a single Image with different arguments instead, it will be more readable.

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (yll090) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: There's no reason to wrap a single element, you can do that in the composable's modifier.

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (yll090) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Please extract the bigger views/screens in separate files, it's more readable and you'll have easier time working on those in the future.

**Review by yll090 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (gbuyukliev-accedia) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Note: Just an advice - always place `modifier` as the first argument. This way you have a better overall visibility of how the composable is positioned at just a glance.

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (gbuyukliev-accedia) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Missing logo image.

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (gbuyukliev-accedia) [inline app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Note: Look into `Scaffold` with `topBar` if you want to achieve this natively. No problem to merge the PR without this.

**Review by gbuyukliev-accedia — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: ok, done

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: done

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Added it

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: ok, sure, I've moved all modifiers on the first lines

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: Extracted the image as variable, as for the description - I just made it generic

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: I'll try to do it when I start implementing HW1-2 and watch the second hour of the lecture

File: app/src/main/java/com/example/androidtrainingproject/MainActivity.kt
Comment (ivanzdravchev-acc) [reply app/src/main/java/com/example/androidtrainingproject/MainActivity.kt]: I wasn't able to center it without it, but now when I got back to it again I figured it out

**Comment by ivanzdravchev-acc** [comment]
Resolved the comments, merging without waiting for another review so I can start work on HW1-2

**Pull request merged**
